dev-perl/Array-Diff/Array-Diff-0.70.0-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
I'm not sure if I've finally finshed doing the required copying of `EAPI6` logic from `::gentoo` to `::perl-experimental`'s `perl-module.eclass` yet, and you may find package managers have problems installing this as a result.

And this is because there was a respectable amount of divergence between `::gentoo` and `::perl-experimental` back in `EAPI5`, and subsequently, reunifying them without breaking things is a bit tricky.